README.markdown

```diff
@@ -34,11 +34,12 @@ Writing Objective-C? Check out our [Objective-C Style Guide](https://github.com/
 
 ## Naming
 
-Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter.
+Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter. Global constants, however, should have a project prefix.
 
 **Preferred:**
 
 ```swift
+let XYZSomeGlobalConstant = 123.456
```

Comment:
@sallyransom Devil's advocate + information gathering: isn't it the job of Swift's modules to avoid name collision? A.K.A. making prefixes unnecessary.